### PR TITLE
Simplify how setup tasks are skipped

### PR DESCRIPTION
**Pull request by rlskoeser**
use `never` to skip setup tasks by default, rather than configuring it as a tag to be skipped by default (which is harder to override)

I haven't run the playbooks with this configuration, but I used the `--list-tasks` option to check and it looks like it's configured properly

### CODE REVIEW

File: roles/deploy_user/tasks/main.yml [hunk 1]
```diff
@@ -4,7 +4,8 @@
   become: true
   tags:
     - setup
-    - cdhsetup
+    - cdhsetup  # ?
```
Comment (acozine) [inline roles/deploy_user/tasks/main.yml:7]: What does this tag do?

**Review by acozine — COMMENTED** [review]
Overall looks great!

File: README.md [hunk 2]
```diff
@@ -77,17 +77,14 @@ The playbook will run, noting success and failures. The `-v` flag adjusts verbos
 
 ### Skip setup tasks
 
-By default, initial provisioning and setup tasks are configured to be skipped.  Tasks or groups of tasks should be tagged as `setup`.
+By default, initial provisioning and setup tasks are configured to be skipped.  Tasks or groups of tasks should be tagged with both `setup` and `never`.
 
 To run playbook without skipping setup tasks, override the default skip tag configuration:
```
[the remark this answers is not in the document]
Comment (rlskoeser) [reply README.md:82]: thank you for catching this!

File: roles/deploy_user/tasks/main.yml (hunk 1 above)
Comment (rlskoeser) [reply roles/deploy_user/tasks/main.yml:7]: I don't think it does anything! I just searched through all our roles and I don't see it used anywhere else. I'll remove

**Pull request merged**